stdlib/universe/range_nsecs_table_test.flux

```diff
@@ -0,0 +1,132 @@
+package universe_test
+
+import "planner"
+import "testing"
+
+option now = () => 2030-01-01T00:00:00Z
+
+// todo(faith): still need to add the following cases--
+// window |> count/sum, window |> first/last, window |> min/max
+// aggregateWindow(count/sum), aggregateWindow(first/last/min/max), group |> first/last
+
+inData = "
+#datatype,string,long,dateTime:RFC3339,string,double,string
+#group,false,false,false,true,false,true
+#default,_result,,,,,
+,result,table,_time,_field,_value,_measurement
+,,0,2021-01-01T00:00:01.000000001Z,foo,1.0,mm
+,,0,2021-01-01T00:00:01.000000002Z,foo,2.0,mm
+,,0,2021-01-01T00:00:01.000000005Z,foo,3.0,mm
+,,0,2021-01-01T00:00:01.000000024Z,foo,11.24,mm
+
+,,1,2021-01-01T00:00:01.000000009Z,bar,4.0,mm
+,,1,2021-01-01T00:00:01.000000011Z,bar,5.0,mm
+,,1,2021-01-01T00:00:01.000000015Z,bar,6.0,mm
+,,1,2021-01-01T00:00:01.000000022Z,bar,1.2,mm
+,,1,2021-01-01T00:00:01.000000031Z,bar,0.28,mm
+"
+
+testcase range_nsecs_bare_last {
+    result = testing.loadStorage(csv: inData)
```

Comment:
Could we refactor this to use `array.from` and `testing.load` rather than `testing.loadStorage`? I think we want to EOL `testing.loadStorage` and it's reliance on csv.

Reply:
Yeah I spent and hour last week trying to use array.from with these tests and switched to csv when i got sufficiently frustrated.

Reply:
I hate writing tests with csv.from as much as the next guy, but when I wrote these tests in array.from initially, I couldn't get them working at all on my own. Maybe we need a guide for the new test suite or something?

Reply:
Its updated now. Lemme know what you think.